var/spack/repos/builtin/packages/r-pkgmaker/package.py

```diff
@@ -16,8 +16,8 @@ class RPkgmaker(RPackage):
     changes in the interface(s) are more than likely to happen."""
 
     homepage = "https://renozao.github.io/pkgmaker"
-    url      = "https://cran.r-project.org/src/contrib/pkgmaker_0.22.tar.gz"
-    list_url = "https://cran.r-project.org/src/contrib/Archive/pkgmaker"
+    url      = "https://cloud.r-project.org/src/contrib/pkgmaker_0.22.tar.gz"
+    list_url = "https://cloud.r-project.org/src/contrib/Archive/pkgmaker"
 
     version('0.27', sha256='17a289d8f596ba5637b07077b3bff22411a2c2263c0b7de59fe848666555ec6a')
     version('0.22', '73a0c6d3e84c6dadf3de7582ef7e88a4')
```

Comment:
Has a bunch of suggested dependencies to remove

Reply:
Done.